README.md

```diff
@@ -1,70 +1,76 @@
-# Getting Started with Create React App
+# COVID_19 CASES TRACKER
+"This application displays Covid-19 confirmed cases from different country in Europe. "
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+![react](https://user-images.githubusercontent.com/57562869/132714311-4af231a9-b3a2-498c-9e2d-ab7e0679a444.png)
 
-## Available Scripts
 
-In the project directory, you can run:
+## Built With
 
-### `npm start`
+- ReactJS
+- Redux
+- Sass
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+## Live demon
 
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
+[See Live](https://determined-archimedes-c0ea66.netlify.app/)
 
-### `npm test`
+## Video 
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+[presentation video](https://www.loom.com/share/83bb11549f314108ab9866f9c78f20e0)
 
-### `npm run build`
+## Getting Started
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+To get a local copy up and running follow these simple example steps.
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+### Setup
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+```bash
+git clone git@github.com:moise10r/covid_19_app.git
+cd ./covid_19_app
+```
 
-### `npm run eject`
+### Install
 
-**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
+```bash
+npm install
+```
 
-If you aren’t satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+### To Run it
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
+```bash
+npm start
+```
```

Comment:
[OPTIONAL]

- I would highly recommend adding the command on how to test the project in the README file. (i.e. `npm test`).

Reply:
Thank you for the feedback. I added the npm run test in my README file.